sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -686,17 +687,19 @@ private[hive] class HiveClientImpl(
       } catch {
         case e: Exception =>
           val remainingParts = matchingParts.toBuffer --= droppedParts
+          // scalastyle:off line.size.limit
           logError(
             s"""
                |======================
-               |Attempt to drop the partition specs in table '$table' database '$db':
+               |Attempt to drop the partition specs in table '${MDC(TABLE_NAME, table)}' database '${MDC(DB_NAME, db)}':
                |${specs.mkString("\n")}
                |In this attempt, the following partitions have been dropped successfully:
                |${droppedParts.mkString("\n")}
                |The remaining partitions have not been dropped:
                |${remainingParts.mkString("\n")}
```

Comment:
Hi, @panbingkun I wonder if you could advise me how can we handle those variables which are made via `mkString`.

They would contains multiple items split by `"\n"`, but can we still just make them as a `LogKey`??

Reply:
Oh, nvm. I found some examples. I think we can just make them as an `LogKey`.

Reply:
yeah

Reply:
No problem. Thanks for your all reviews!